owncloud/ios-app
# [fix/single-accouont-display-name] Show Display Name in Branded Single Account View

## Description
Username could look like a UUID, which is not user friendly. Now showing the `displayName`, if available, other show the `userName`.

## Related Issue
https://github.com/owncloud/enterprise/issues/4798

## Motivation and Context


## How Has This Been Tested?
- brand app as oc.online
- configure an account
- check header for `displayName`

## Screenshots (if appropriate):

## Types of changes


## Checklist:
